File: 6) C E C++/retangulo.cpp
#include <iostream>
#include <cmath>

int main( ) {
	float area, base, altura, perimetro, diagonal;
	
	std::cout << "Digite o valor da base: ";
	std::cin >> base;
	std::cout << "Digite o valor da altura: ";
	std::cin >> altura;
	
	area = base * altura;
	perimetro = 2 * ( base + altura );
	diagonal = sqrt( ( base * base ) + ( altura * altura ) );
	
	std::cout << "Area: " << area << "\nPerimetro: " << perimetro << "\nDiagonal: " << diagonal;
}

File: 4) C#/retangulo.cs
using System;

namespace Retangulo 
{
	class Program
	{
		static void Main( string[ ] args )
		{
			float a, b, area, perimetro;
			double diagonal;
			
			Console.WriteLine( "Digite a altura do retangulo: " );
			a = float.Parse( Console.ReadLine( ) );
			Console.WriteLine( "Digite a base do retangulo: " );
			b = float.Parse( Console.ReadLine( ) );
			
			area = a * b;
			perimetro = 2 * ( b + a );
			diagonal = Math.Sqrt( ( b * b ) + ( a * a ) );
			
			Console.WriteLine( $"Area: {area}\nDiagonal: {diagonal}\nPerimetro: {perimetro}" );
		}
	}
}